src/sdk/github.js

```diff
@@ -12,7 +12,7 @@ let githubAppId
 let githubAuth
 
 // Load fetch polyfill for browsers not supporting fetch API
-if (!window.fetch) {
+if (typeof window !== 'undefined' && !window.fetch) {
```

Comment:
Actually, `window.fetch` is required by `github` provider to get user info (see [line 99](https://github.com/deepakaggarwal7/react-social-login/blob/master/src/sdk/github.js#L99)) and to get access token (see [line 167](https://github.com/deepakaggarwal7/react-social-login/blob/master/src/sdk/github.js#L167)).

Furthermore, `window` is used by `amazon` ([`window.amazon`](https://github.com/deepakaggarwal7/react-social-login/blob/master/src/sdk/amazon.js#L41)), `facebook` ([`window.FB`](https://github.com/deepakaggarwal7/react-social-login/blob/master/src/sdk/facebook.js#L43)), `google` ([`window.gapi`](https://github.com/deepakaggarwal7/react-social-login/blob/master/src/sdk/google.js#L19)) and `linkedin` ([`window.IN`](https://github.com/deepakaggarwal7/react-social-login/blob/master/src/sdk/linkedin.js#L22)) providers... So if `window` is undefined, it will not work anyway...

But maybe I don’t get what you want to do at all?

Reply:
Hi,

You are right, but this won't have any effect on the browser side, as `window` will be defined.

This is merely a fix to allow `react-social-login` module to be used in a server side rendered react app.
It won't make `react-social-login` work server side, as this would make no sense with these client side authentications, but will allow to have the same code working, client or server side rendered.

Without it, the following error is thrown by node :
```
/node_modules/react-social-login/dist/social-login.js:2077
if (!window.fetch) {
^

ReferenceError: window is not defined
    at Object.defineProperty.value (/node_modules/react-social-login/dist/social-login.js:2077:1)
    at __webpack_require__ (/node_modules/react-social-login/dist/social-login.js:30:30)
    at Object.<anonymous> (/node_modules/react-social-login/dist/social-login.js:1086:15)
    at __webpack_require__ (/node_modules/react-social-login/dist/social-login.js:30:30)
    at Object.defineProperty.value (/node_modules/react-social-login/dist/social-login.js:1517:12)
    at __webpack_require__ (/node_modules/react-social-login/dist/social-login.js:30:30)
    at Object.defineProperty.value (/node_modules/react-social-login/dist/social-login.js:1245:18)
    at __webpack_require__ (/node_modules/react-social-login/dist/social-login.js:30:30)
    at Object.<anonymous> (/node_modules/react-social-login/dist/social-login.js:3695:18)
    at __webpack_require__ (node_modules/react-social-login/dist/social-login.js:30:30)
```

I believe this may fix #39 too (although I have not tested with next.js). Furthermore it avoids dynamic loading of the module, which is making the importing app and `react-social-login` be tightly coupled and adds complexity.